### DISCUSSION TITLE
Absolute URLs preventing wiki.js from being accessible from a subfolder?

### DISCUSSION THREAD

**Discussion by Greg-J** [requarks/wiki#4032] [Help / Questions] [answered]
I followed the instructions and have installed wiki.js successfully and can access it on my local IP of `http://192.168.200.100:3000/` just fine.

However, when I try to access it from my reverse proxy I get timeouts on all of the assets because it's trying to access them from domain.com/_assets however I have this pulling from domain.com/wiki. The host key in the db shows `https://dekocraft.com/wiki/` but for whatever reason this config isn't respected.

Why is wiki.js using absolute paths instead of relative paths and how do I install wiki.js in a subfolder?

**Comment by Phil-hostwiki** [comment, marked as answer]
Wiki.js does not support subfolder installations.

https://feedback.js.wiki/wiki/p/support-hosting-wikijs-at-subpath

**Reply by Greg-J** [reply]
But of an oversight there, innit?
